client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/pattern-list-renderer.scss

```diff
@@ -1,3 +1,27 @@
 .pattern-list-renderer__pattern-list-item {
 	padding: 0;
+	position: relative;
+	overflow: visible;
+
+	.components-tooltip {
+		top: 100% !important;
+		left: 0 !important;
+		margin-top: 5px;
+		height: min-content;
+
+		.components-popover__content {
+			min-width: 100% !important;
+			white-space: normal;
+		}
+	}
+
+	&:last-child {
+		.components-tooltip {
+			top: unset !important;
+			bottom: 100% !important;
```

Comment:
I think we can add padding-bottom to the scroll container instead of changing the position. WDYT?

Reply:
Thanks. It didn't work and I had to change the position.